README.md

```diff
@@ -3,11 +3,12 @@
 Live Site (version 1): https://goodgovproject.com/
```

Comment:
maybe add  that this Live Site was for the previous project

Reply:
good idea!